plugins/community/salesforce-reactive-grpc-java/v1.2.4/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM adoptopenjdk/openjdk11:jre-11.0.20_8-debianslim
```

Comment:
A few notes on the Dockerfile

- Ideally, we use one of the base images tracked in https://github.com/bufbuild/plugins/tree/main/.github/docker
- Can the resulting plugin be used in a distroless image
- We typically use `USER nobody`
- Avoid using ARG and hard code the name

Reply:
Unfortunately the plugin cannot be used in a distroless image as the native executable is a [golang canteen launcher](https://github.com/salesforce/grpc-java-contrib/tree/master/canteen) which has a jar concatenated on the end of it. A JVM is still required in the image. Other issues resolved.